Models.py

```diff
@@ -235,9 +300,16 @@ def update_cached_offline(self):
 
     def getInfoToday(self, period: int=14) -> List[float]:
         """
-        Similar to getHistoricalData but it only gets today and the last relevant days to the stock
+        This method will get the information for the stock today and the last relevant days to the stock.
+        
+        The cached_data is used so less data has to be retrieved from yf.finance as it is already being
+        held and is quicker to obtain.
+
+        Args:
+            period (int): The number of days to get the information for
         
-        cached_data used so less data has to be gotten from yf.finance
+        Returns:
+            stock_data[key].values.tolist() (List[float]): The information for the stock today and the last relevant days to the stock
```

Comment:
ChatGpt says it should be `list:` instead of `stock_data[key].values.tolist() (List[float])`